auraed/Cargo.toml

```diff
@@ -64,9 +64,10 @@ rtnetlink = "0.11.0"
 netlink-packet-route = "0.13.0" # Used for netlink_packet_route::rtnl::address::nlas definition
 thiserror = "1.0.37"
 crossbeam = "0.8"
-libcontainer     = { git = "https://github.com/containers/youki.git", tag = "v0.0.2" }
-liboci-cli     = { git = "https://github.com/containers/youki.git", tag = "v0.0.2" }
-
+libcontainer = { git = "https://github.com/containers/youki.git", tag = "v0.0.2" }
```

Comment:
why do we depend on this same repo twice with different names?

Reply:
When you rely on a git repo, cargo will look for a crate with the name you use, in that repo (not the whole repo). So here, there are 2 different crates being used that happen to be in the same repo. [Docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories)

Reply:
ah thanks :) #til